### PR TITLE
created .htaccess

### PR DESCRIPTION
**Pull request by ppquadrat**
Hi! Please add the Jazz Ontology

### CODE REVIEW

**Comment by ppquadrat** [comment]
Thanks!

File: JazzOntology/.htaccess [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://github.com/ppquadrat/JazzOntology/blob/master/JazzOntology.owl [R=302,L]
```
Comment (davidlehn) [inline JazzOntology/.htaccess]: Did you mean to link to a raw file or the github ui?

**Comment by ppquadrat** [comment]
Hi,
I wanted to link to the owl file. Is it the right way to do it?
All the best,
Polina

**Comment by dgarijo** [comment]
@ppquadrat 
No. You should either create a github page or use the links from github to serve the raw content. something like https://raw.githubusercontent.com/ppquadrat/JazzOntology/master/JazzOntology.owl

**Comment by ppquadrat** [comment]
Many thanks. Shall I correct it or would you change the link for me please?
Polina

**Comment by dgarijo** [comment]
Please go ahead and correct it :)

**Pull request merged**